installer/roles/ansible-service-broker-setup/defaults/main.yml

```diff
@@ -5,4 +5,5 @@ dockerhub_tag: latest
 dockerhost: docker.io
 launch_apb_on_bind: true
 skip_apb: 
-apb_sync: false
\ No newline at end of file
+apb_sync: false
+ansible-service-broker: ansible-service-broker
```

Comment:
[Minor] It's missing what is that related to. In this case namespace. 
If that's not the problem It may be worth to change `ansible-service-broker` => `ansible-service-broker-namespace`